content/blog/2023/when-to-choose-a-responsive-web-app.md

```diff
@@ -0,0 +1,160 @@
+
+RWAs take less time to build, relative to native mobile apps; it only takes one codebase to reach any device with a browser. It can also take less time to make the minimum viable product (MVP) available to an initial set of users, get feedback, and release frequent, incremental updates.
+
+Another caveat. No matter the type, even a very simple MVP of your app will take at least a few months to build. Very complex apps often take multiple years to design and develop. If you want to make sure you're meeting the needs of the people who use your app, you will want to allow time to collect their feedback along the way and make improvements.
+
+### Less Maintenance
+*…when you need to keep ongoing maintenance costs low.*
+
+RWAs require less time and expense to maintain compared to native mobile apps. An RWA’s single codebase is built with standard web technologies, and doesn’t require an update of the entire app to make incremental improvements. It also takes less time and fewer developers to update a single codebase than to update separate codebases that use specialized technology for multiple platforms.
+
+### Discoverable Via Search Engine
+*…when the people you want to reach will look for your content with a search engine.*
+
+Search engines, like Google, crawl the web looking for keywords and other information to sort and rank the content. Then they collect, process, and store a database of all that content. This is called *indexing*. RWAs live on the web, so unless special code is added to prevent it, they will automatically be indexed and appear in organic search results.
+
+### Broad Reach
+*…when you want to reach a broad audience across devices.*
+
+People can access the same RWA on any device from a small iPhone to a tablet, from a laptop to a giant Windows desktop, unlike native mobile apps which are limited to supported mobile devices. Additionally, the web provides a high level of backward compatibility, meaning RWAs can be available in some form even on old devices. On the other hand, one platform update can totally block a device from accessing a native mobile app.
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2023/no.png',
+    alt: 'The word no with a frowning face'
+  }],
+  attrs={
+    'style': '--column: content-start'
+  }
+) }}
+
+## Don’t choose an RWA when…
+### Limited Capabilities
+*…when your project requires a high level of specialized capability.*
+
+RWAs cannot access all of a device’s hardware and software features. For example, if your digital product requires direct integration with a specific mobile device’s GPS for geofencing or access to light and proximity sensors, then an RWA won’t work.
+
+It is *possible* to access some advanced device features in the context of a RWA, but then you would be moving into the [progressive web app](/2023/07/05/when-to-choose-a-progressive-web-app/) space, not strictly an RWA which provides static content for people to consume.
```

Comment:
> which provides static content for people to consume.

That seems overly narrow. Most of the apps we build for clients use the RWA approach, and almost none of them are 'static content for people to consume'. You don't need access to device features in order to have dynamic content.